gef.py

```diff
@@ -2011,10 +2011,10 @@ def gdb_get_location_from_symbol(address: int) -> Optional[Tuple[str, int]]:
         return None
 
     i = sym.find(" in section ")
-    sym = sym[:i].split()
+    sym = sym[:i].split('+')
     name, offset = sym[0], 0
-    if len(sym) == 3 and sym[2].isdigit():
-        offset = int(sym[2])
+    if len(sym) == 2 and sym[1].isdigit():
```

Comment:
What does this logic do? Why did the indices change? Could you please add a comment?

Reply:
this is because the c symbol is essentially "<name (no space)> + <offset>", and we are splitting the string by space. hence [0] = symbol, [1] = "+", [2] = offset. In C++, symbol names can have space in this, hence broken.

a more proper way is simply split by "+".

Reply:
Maybe we should switch this to regex? If you think this is robust as is, that's fine too, but let's add a comment to clarify since I don't want to have to figure it out again by random indices :) Maybe show a sample symbol name before the "in section" split.